src/Stratis.Bitcoin/Base/TimeSyncBehavior.cs

```diff
@@ -167,6 +173,9 @@ public struct TimestampOffsetSample
         /// otherwise the node may not perform correctly on the network.</summary>
         private IAsyncLoop warningLoop;
 
+        /// <summary> A value indicating whether the system time is not in sync and needs adjustment. </summary>
+        private bool isSystemTimeOutOfSync;
```

Comment:
why not using `WarningLoopStarted`  that is already there doing exactly this?

Reply:
While they're being set up at the same time, I don't like to rely on a flag showing something that's a result of a certain state. 
Here, the system is out of sync, therefore the loop starts. I don't want to look at whether the loop has started to conclude that we're out of sync. I'd rather have another flag that explicitly tells me that.

Reply:
Then you should probably just rename `WarningLoopStarted` to `IsSystemTimeOutOfSync`, Because that was the purpose of that property, now we have 2 things doing absolutely the same, and one is not used, so maybe just remove it

Reply:
Good idea :-)